docs/src/development/local/ddev.md

```diff
@@ -84,16 +78,17 @@ ddev config --web-environment-add="PLATFORM_PROJECT={{< variable "PROJECT_ID" >}
 For example:
 
 ```bash
-ddev config --web-environment-add="PLATFORM_PROJECT=nf36mudfdd23bi,PLATFORM_ENVIRONMENT=main"
+ddev config --web-environment-add="PLATFORM_PROJECT=abcdefgh1234567,PLATFORM_ENVIRONMENT=main"
 ```
 
-Get DDEV to recognize the variables by restarting:
+The best way to connect your local DDEV to your Platform.sh project is through the [Platform.sh DDEV add-on](https://github.com/platformsh/ddev-platformsh).
+Add it by running the following command:
```

Comment:
💅 

```suggestion
To add it, run the following command:
```

Reply:
We should make it clear that it's for PHP projects only (well, DDEV is PHP only anyway, at least for now).